src/GraphNodeManagerViewExtension/GraphNodeManagerViewModel.cs

```diff
@@ -206,18 +206,18 @@ public GraphNodeManagerViewModel(ViewLoadedParams p, string id, Action<Logging.I
 
 
         private void InitializeFilters()
-        {   //TODO this should really not use localized text... it should use an enum etc.
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_EmptyList }); 
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_Error });
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_Frozen });
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_MissingContent });
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_Function });
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_Information });
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_IsInput });
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_IsOutput });
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_Null });
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_Warning });
-            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_PreviewOff });
+        {   
+            FilterItems.Add(new FilterViewModel(this){Name = Resources.Title_EmptyList, FilterImage = "pack://application:,,,/GraphNodeManagerViewExtension;component/Images/EmptyList.png" }); 
```

Comment:
I'm not plugged in enough to be in the know here -but I will say, it's easier when all image loading is done in the same way - is this standard throughout the code base and other extensions?

Reply:
Giving the string path to an image resource in the code-behind is used in the code base. I think I might not be fully getting your point here maybe? https://github.com/DynamoDS/Dynamo/blob/36c513d9f4e75b4a154467e17df389a2a5d64eab/src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs#L551